src/js/modules/core/components/MastheadBar.js

```diff
@@ -34,22 +33,58 @@ const styles = {
     fontSize: '26px',
     marginTop: '4px',
     textDecoration: 'none',
+    '&:hover': {
+      color: '#000',
+      textDecoration: 'none',
+    },
+    '&:focus': {
+      color: '#000',
+      textDecoration: 'none',
+    },
   },
   userTools: {
     float: 'right',
     marginTop: '9px',
     '& button': {
       marginLeft: '24px',
     }
+  },
+};
+
+const navButtonStyles = {
+  NavButton: {
+    background: 'none',
+    borderWidth: 0,
+    margin: 0,
+    padding: 0,
+    '&:hover': {
+      cursor: 'pointer',
+    }
+  },
+  buttonText: {
+    color: '#000',
+    fontFamily: 'Circular Std',
+    fontSize: '12px',
+    fontWeight: 'bold',
+    textTransform: 'uppercase',
   }
 };
 
+const UnstyledNavButton = ({ classes, label }) => {
```

Comment:
little awkward. I'd just make a separate file or just call the second one StyledNavButton